main.rb

```diff
@@ -0,0 +1,30 @@
+def decode_char(char)
+  # rubocop:disable Layout/LineLength
```

Comment:
[OPTIONAL]
- [ ] kindly make sure you are not disabling the linters checks for they help your code to fit the standards norms.

`Hint: To avoid this error you may consider writing your hash this way:`
```rb
morse_code = {
  '.-' => 'a',
  '-...' => 'b',
  '-.-.' => 'c,
# ...
}
```